defaults/main.yml

```diff
@@ -22,6 +22,19 @@ apache_additional_config_folders:
   - sites-enabled
   - conf-enabled
 
+# mod_jk variables
+apache_modjk_install: true
+apache_modjk_prerrequisites:
+  - autoconf
+  - libtool-bin
+apache_modjk_url: http://apache.rediris.es/tomcat/tomcat-connectors/jk/tomcat-connectors-{{ apache_modjk_version }}-src.tar.gz
```

Comment:
Personally, I would prefer that variable on _vars/main.yml_

Reply:
Done